portal-impl/src/com/liferay/mail/service/impl/MailServiceImpl.java

```diff
@@ -246,7 +248,22 @@ public Session getSession() {
 			}
 		}
 
-		_session = Session.getInstance(properties);
+		if (smtpAuth) {
```

Comment:
Shouldn't we do this only when the protocol is configured to be SMTPS? It is possible to have smtpAuth = true with regular SMTP (i.e. no TLS).

Reply:
Based on the results of my testing, setting this param to true is necessary for the mail to be sent, when the protocol is regular SMTP.
BTW, if the protocol is SMTPS, the sending fails with this error
```
2020-03-13 15:03:08.882 ERROR [liferay/mail-4][MailEngine:128] Unable to send message: Exception reading response
javax.net.ssl.SSLException: Unrecognized SSL message, plaintext connection?
```

Reply:
You're right. Sorry for the misunderstanding.